operator-framework-core/src/main/java/io/javaoperatorsdk/operator/Metrics.java

```diff
@@ -29,7 +29,10 @@ public Metrics(MeterRegistry registry) {
       Context<R> context) {
     final var name = configuration.getName();
     final var timer =
-        registry.timer("operator.sdk.controllers.execution.createOrUpdate", "controller", name);
+        Timer.builder("Controller Executions Timer")
```

Comment:
Why was only this timer changed? What happened to the name and the tags? Also, please strive to create atomic, functional commits. A commit should stand by itself with a single functionality added / implemented as much as possible. As this commits stands, it's missing the delete counterpart which should be part of the same commit.

Reply:
Yes, I will add same naming conventions and update it for Delete method as well.